ignite/cmd/plugin_default.go

```diff
@@ -18,6 +18,8 @@ type defaultPlugin struct {
 const (
 	PluginNetworkVersion = "v0.2.2"
 	PluginNetworkPath    = "github.com/ignite/cli-plugin-network@" + PluginNetworkVersion
+	PluginRelayerVersion = "hermes/v0.2.4"
```

Comment:
that version doesn't exist yet. Shall we use a commit for now?

Reply:
We may want https://github.com/ignite/apps/pull/111 before tagging v0.2.4 and it needs v28.4.1 beforehand to be able to do so.

Reply:
I need to release the version `hermes/v0.2.4` based on https://github.com/ignite/apps/pull/111 because we need to remove the relayer command from CLI and use it inside the Hermes app. If not the app will break

Reply:
But I think I will remove the default apps now since the network is not working and the hermes needs the cycle dependency

Reply:
I think, so there will be like only 10 min where it won't work I guess. Makes sense.